2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
I'm not able to get rid of jiggling entirely - this may be because I have a dataset with non-unique index (aka date). Very small jiggling is visible even after sorting.

![cum_enrolment_workshops](https://cloud.githubusercontent.com/assets/72821/6812998/939a8396-d272-11e4-9106-856142017420.png)

![cum_workshops](https://cloud.githubusercontent.com/assets/72821/6813188/dd460852-d273-11e4-8663-314ffcce998c.png)

![cum_instructors](https://cloud.githubusercontent.com/assets/72821/6813096/316c720a-d273-11e4-8673-83d06515cbb4.png)
(To fix horrible jiggling in this one I removed entries with duplicated dates leaving the latest entry with biggest number of instructors)

Anyway, I'll include two nicest looking plots I could get to this proposal and later change them to links - if they get published. Otherwise I'll link to the plot you pointed out earlier (https://github.com/numfocus/gsoc/pull/36#discussion_r27068426).